elemnets/textInput.js

```diff
@@ -10,6 +11,15 @@ class WebTextInput extends WebElement{
         console.log('Clearing field');
         await this.protractorElement.clear();
     }
+
+    async check() {
+        console.log('Checking checkbox');
+        let checked = await this.protractorElement.isSelected();
```

Comment:
Please test this method if it works correctly in the case with already selected checkbox

Reply:
move to checkbox file

Reply:
Done